pkg/login/social/github_oauth.go

```diff
@@ -47,8 +53,13 @@ func (s *SocialGithub) IsTeamMember(client *http.Client) bool {
 		return false
 	}
 
+	teamMembershipIds := make([]int, len(teamMemberships))
+	for i, team := range teamMemberships {
+		teamMembershipIds[i] = team.Id
+	}
+
 	for _, teamId := range s.teamIds {
-		for _, membershipId := range teamMemberships {
+		for _, membershipId := range teamMembershipIds {
```

Comment:
Any reason for creating the second array vs just:
```suggestion
		for _, membership := range teamMemberships {
		  if teamId == membership.Id {
```

Reply:
Yea, good point! I made rough and dirty POC, so refactor is required.